plugins/com.google.cloud.tools.eclipse.appengine.ui/plugin.xml

```diff
@@ -58,18 +58,6 @@
                  style="push">
               <parameter
                     name="newWizardId"
```

Comment:
Removing the Maven-based std wizard (along with the separator between the std wizards and the flex wizard) from the GCP toolbar menu.